universal-application-tool-0.0.1/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -80,18 +89,67 @@ public Content render(
     return layout.renderWithNav(request, userName, messages, bundle);
   }
 
-  private ContainerTag renderTIApplicantsTable(TrustedIntermediaryGroup tiGroup) {
-    return div(
-        table()
-            .withClasses(Styles.BORDER, Styles.BORDER_GRAY_300, Styles.SHADOW_MD, Styles.W_3_4)
-            .with(renderApplicantTableHeader())
-            .with(
-                tbody(
-                    each(
-                        tiGroup.getManagedAccounts().stream()
-                            .sorted(Comparator.comparing(Account::getApplicantName))
-                            .collect(Collectors.toList()),
-                        account -> renderApplicantRow(account)))));
+  private ContainerTag renderSearchForm(Http.Request request, Optional<String> search) {
+    return form()
+        .withMethod("GET")
+        .withAction(
+            routes.TrustedIntermediaryController.dashboard(Optional.empty(), Optional.empty())
+                .url())
+        .with(
+            FieldWithLabel.input()
+                .setId("search-field")
+                .setFieldName("search")
+                .setLabelText("Search")
+                .setValue(search.orElse(""))
+                .setPlaceholderText("Search")
+                .getContainer()
+                .withClasses(Styles.W_1_4),
+            makeCsrfTokenInputTag(request),
+            submitButton("Search").withClasses(Styles.M_2));
+  }
+
+  private ContainerTag renderTIApplicantsTable(
+      ImmutableList<Account> managedAccounts,
+      Optional<String> search,
+      int page,
+      int totalPageCount) {
+    ContainerTag main =
+        div(table()
+                .withClasses(Styles.BORDER, Styles.BORDER_GRAY_300, Styles.SHADOW_MD, Styles.W_3_4)
+                .with(renderApplicantTableHeader())
+                .with(
+                    tbody(
+                        each(
+                            managedAccounts.stream()
+                                .sorted(Comparator.comparing(Account::getApplicantName))
+                                .collect(Collectors.toList()),
+                            account -> renderApplicantRow(account)))))
+            .withClasses(Styles.MB_16);
+    ContainerTag div = div();
+    if (page <= 1) {
+      div.with(new LinkElement().setText("∅").asButton());
+    } else {
```

Comment:
What does the `∅` button do? Perhaps just omit next/previous buttons if it doesn't apply? 

```suggestion
    if (page > 1) {
```

If you want to keep the button so that buttons are in a consistent location I understand that, too, but then we should just preserve the **←** text and disable the buttons. (We can do that in a follow-up if you don't want to bother with it now since we don't have disabled button styling yet - should be coming soon.)

Reply:
Yep, just to keep things in the same place - but the buttons are actually links, right?  They're not `button`s, they're `a`s.  And disabling a "button" that is actually a link is super annoying / hard.  Once we get disabled button styling we can do that.